controllers/utils-functions.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+const format = require("pg-format");
+
+exports.checkExists = async (table, column, value) => {
```

Comment:
Love that you're using this, but you have not tested it in any way that I can see. All functions should be tested to prove they work before you start using them in other functions to give you the confidence they will do what you want.